src/fiskaltrust.Launcher/ProcessHost/ProcessHostMonarch.cs

```diff
@@ -88,6 +94,23 @@ public Task Start(CancellationToken cancellationToken)
             {
                 _logger.LogInformation("Host {package} {id} has shutdown.", _packageConfiguration.Package, _packageConfiguration.Id);
 
+                if (_process.ExitCode != 0)
+                {
+                    foreach (var log in _plebianLogBuffer)
+                    {
+                        if (log.Contains("Error"))
+                        {
+                            _logger.LogError(log);
+                        }
+                        else
+                        {
+                            _logger.LogInformation(log);
+                        }
+                    }
+                }
+
+                _plebianLogBuffer.Clear();
```

Comment:
We need to move this logic down to line 120 into the try catch block.

Because with the `if (!cancellationToken.IsCancellationRequested)` in line 115 we make sure this is not done when shut down the launcher with a `ctrl+c`.